scripts/config/templates/nginx.conf.erb

```diff
@@ -90,11 +90,15 @@ http {
     }
   <% end %>
 
+  location ~ ^.*(static|assets|images)\/.*\.(jpg|jpeg|png|gif|ico|css|js|jsx|ts|tsx|woff|woff2|ttf|otf|eot)$ {}
```

Comment:
Doing a pattern match on these files excludes them for being matched in the routes block where `etag` and `Last-Modified` headers are disabled.